script/repos.js

```diff
@@ -0,0 +1,13 @@
+var repos = {};
+
+repos.all = [];
+
+repos.requestAll = function(callback) {
+  $.ajax({
+    type: 'GET',
+    url: 'https://api.github.com/users/tesmonrd/repos' + '?sort=updated&per_page=100',
+    headers: { Authorization: 'token ' + githubToken}
+  }).done(function(data){
+    repos.all = data;
+  }).done(callback);
+};
```

Comment:
Consider adding a .fail() method here. That way your users will be notified as to why the about button isn't working. Currently nothing at all happens if this fails. This could happen if github is down, for example.